app/migration/mappers/training_programme_type_mapper.rb

```diff
@@ -0,0 +1,19 @@
+module Mappers
+  class TrainingProgrammeTypeMapper
+    # Maps the ECF1 values to the ECF2 values
+    MAPPING = {
+      "core_induction_programme" => "school_led",
+      "design_our_own" => "school_led",
+      "school_funded_fip" => "provider_led",
+      "full_induction_programme" => "provider_led"
+    }.freeze
```

Comment:
Just a question for my own understanding: 

Is `training_programme` on `InductionProgramme` from ECF1 equivalent to `programme_type` on `ECTAtSchoolPeriod` on ECF2?

Reply:
@avinhurry correct